site/config.toml

```diff
@@ -23,3 +23,12 @@ theme           = "github-project-landing-page"
 
     header_link_color="#777"
     header_link_hover_color="rgb(51, 51, 51)"
+
+[params.social]
+  Github        = "search?q=topic%3Areuse+org%3Afsfe&type=Repositories"
```

Comment:
Does this work?

Reply:
Yes, at least in my local demo. it's basically just appending this to `https://github.com/`, see the `social.html` partial L8